core/container/pom.xml

```diff
@@ -25,9 +25,9 @@
   <properties>
     <swarm.fraction.internal>true</swarm.fraction.internal>
     <swarm.fraction.stability>stable</swarm.fraction.stability>
-    <version.weld-api>3.0.Alpha19</version.weld-api>
-    <version.weld>3.0.0.Alpha16</version.weld>
-    <version.cdi2>2.0.Alpha4</version.cdi2>
+    <version.weld-api>2.3.SP2</version.weld-api>
+    <version.weld>2.3.5.Final</version.weld>
+    <version.cdi>1.2</version.cdi>
```

Comment:
I am not very familiar with WF feature packs. Doest it mean to be picked up from WF parent? Unfortunately there is no `weld-se-core` dependency in the WF parent so I guess we need to define it.